documentation/book/proc-resizing-persistent-volumes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-resizing-persistent-volumes-{context}']
+= Resizing persistent volumes
+
+This procedure describes how to increase the size of persistent volumes used by existing {ProductName} cluster.
+This procedure can be applied to clusters using a single persistent volumes as well as to clusters using multiple persistent volumes in a JBOD storage.
+
+NOTE: {ProductPlatformName} currently supports only increasing size of persistent volumes.
+Shrinking of volumes is currently not possible.
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with support for volume resizing
+* A running Cluster Operator
+* A storage class which supports volume expansion
```

Comment:
Should we explain how the user might find this out?

Reply:
There is actually not so much docs about this in Kubernetes. I guess their modular documentation also got them to document stuff as blog posts, so this is probably the best information: https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/

I can reference it there if you think it is worth it. But I think the topic is too complex to write this into our on docs.

WDYT?

Reply:
I agree we should not try to document this ourselves. I think linking to the blog post is going to be helpful. It might be better suited to the "See also" section at the end of the module.

Reply:
Oki, done.